src/TestingUtils/Microsoft.AspNetCore.Testing/src/FlakyOn.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    public static class FlakyOn
+    {
+        public const string All = "All";
+
+        public static class Helix
+        {
+            public const string All = QueuePrefix + "All";
+
+            public const string Fedora28Amd64 = QueuePrefix + HelixQueues.Fedora28Amd64;
+            public const string Fedora27Amd64 = QueuePrefix + HelixQueues.Fedora27Amd64;
+            public const string Redhat7Amd64 = QueuePrefix + HelixQueues.Redhat7Amd64;
+            public const string Debian9Amd64 = QueuePrefix + HelixQueues.Debian9Amd64;
+            public const string Debian8Amd64 = QueuePrefix + HelixQueues.Debian8Amd64;
+            public const string Centos7Amd64 = QueuePrefix + HelixQueues.Centos7Amd64;
+            public const string Ubuntu1604Amd64 = QueuePrefix + HelixQueues.Ubuntu1604Amd64;
+            public const string Ubuntu1810Amd64 = QueuePrefix + HelixQueues.Ubuntu1810Amd64;
+            public const string macOS1012Amd64 = QueuePrefix + HelixQueues.macOS1012Amd64;
```

Comment:
nit: macOS gets the special lowercase treatment? :)

Reply:
I mean... it's the name of the OS (macOS). I definitely spent a non-zero amount of time fretting over that ;)